### ISSUE TITLE
Update Hindi translations

### ISSUE THREAD

**Issue by AninditaBasu** [squidfunk/mkdocs-material#4945]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "hi",
  "action.edit": "इस पृष्ठ को संपादित करें",
  "action.skip": "विषय पर बढ़ें",
  "action.view": "इस पृष्ठ का सूत्र देखें ⬅️",
  "announce.dismiss": "इसे पुनः न दिखायें ⬅️",
  "blog.archive": "पुरालेख ⬅️",
  "blog.categories": "वर्ग ⬅️",
  "blog.categories.in": "में ⬅️",
  "blog.continue": "पढ़ते रहिये ⬅️",
  "blog.draft": "मसौदा ⬅️",
  "blog.index": "सूचि को लौटें ⬅️",
  "blog.meta": "मेटाडेटा ⬅️",
  "blog.references": "सम्बंधित लिंक ⬅️",
  "clipboard.copy": "क्लिपबोर्ड पर कॉपी करें",
  "clipboard.copied": "क्लिपबोर्ड पर कॉपी कर दिया गया",
  "consent.accept": "स्वीकार करें ⬅️",
  "consent.manage": "सेटिंग्स मैनेज करें ⬅️",
  "consent.reject": "अस्वीकार करें  ⬅️",
  "footer": "फुटर ⬅️",
  "footer.next": "आगामी",
  "footer.previous": "पिछला",
  "header": "Header ⬅️",
  "meta.comments": "टिप्पणियाँ",
  "meta.source": "स्रोत",
  "nav": "Navigation ⬅️",
  "readtime.one": "1 मिनट पढ़ने को ⬅️",
  "readtime.other": "# मिनट पढ़ने को ⬅️",
  "rss.created": "RSS फीड ⬅️",
  "rss.updated": "नवीनतम विषयवस्तु का RSS feed ⬅️",
  "search": "खोजें ⬅️",
  "search.placeholder": "खोज",
  "search.share": "शेयर करें ⬅️",
  "search.reset": "हटा दें ⬅️",
  "search.result.initializer": "खोज शुरू करें ⬅️",
  "search.result.placeholder": "खोज शुरू करने के लिए टाइप करें",
  "search.result.none": "कोई मिलान डॉक्यूमेंट नहीं",
  "search.result.one": "1 मिलान डॉक्यूमेंट",
  "search.result.other": "# मिलान डाक्यूमेंट्स",
  "search.result.more.one": "1 और इस पृष्ठ पर ⬅️",
  "search.result.more.other": "# और इस पृष्ठ पर ⬅️",
  "search.result.term.missing": "ग़ायब ⬅️",
  "select.language": "भाषा चुनें ⬅️",
  "select.version": "वर्शन चुनें ⬅️",
  "source": "रिपॉजिटरी पर जाएं",
  "source.file.contributors": "योगदाता ⬅️",
  "source.file.date.created": "बनाया था",
  "source.file.date.updated": "आखिरी अपडेट",
  "tabs": "टैब ⬅️",
  "toc": "विषय - सूची",
  "top": "शीर्षभाग पर लौटें ⬅️"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Perfect, thanks! Added in 6c8c6155c 😊

**Comment by squidfunk** [comment]
Released as part of [9.0.9](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.9)
